goldmane/pkg/aggregator/bucketing/bucket_ring.go

```diff
@@ -26,6 +27,8 @@ import (
 	"github.com/projectcalico/calico/libcalico-go/lib/set"
 )
 
+var StopBucketIteration = errors.New("stop bucket iteration")
```

Comment:
Comment for future work, would be nice to have a single constant in a collection package that represents short circuiting iterators.

Actually wondering if we should switch to using `iter.Seq` for our iterators in the future, it handles short circuiting.